src/pages/jisun/Main/Main.jsx

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import "./Main.scss";
+import { Link, useNavigate } from "react-router-dom";
+import "../../../style/reset.scss";
+
+const Main = () => {
+  const navigate = useNavigate();
+
+  const goToMain = () => {
+    navigate("/jisun-main");
+  };
+
+  return (
+    <div className="main">
+      <nav>
+        <Link to="/jisun-main" className="goHome">
+          <span className="logo"></span>
+          <p>Westagram</p>
+        </Link>
+        <div className="inputWrap">
+          <input type="search" placeholder="검색" />
+        </div>
+        <div className="icons">
+          <a href="" className="icon1"></a>
+          <a href="" className="icon2 on"></a>
+          <a href="" className="icon3"></a>
+        </div>
+      </nav>
+      <div className="contentsWrap">
+        <div className="feeds">
+          <article>
+            <div className="accountWrap">
+              <a className="accountPicture">
+                <img alt="계정 프로필 사진" src="../../../../images/jisun/img_profile.png" />
```

Comment:
해당 상대경로가 public폴더를 참고하는거라면
```suggestion
                <img alt="계정 프로필 사진" src="/images/jisun/img_profile.png" />
```
이렇게 표현할 수 있습니다